### ISSUE TITLE
Get the result after executing the UPDATE SQL request

### ISSUE THREAD

**Issue by AlexanderZhirov** [adamdruppe/arsd#325]
I am making a request to update an entry in PostgreSQL. I would like to get the result to understand whether the request was executed or not.

![изображение](https://user-images.githubusercontent.com/37277056/167300062-2f44844b-2643-4fc2-bc8b-2be5521f9e31.png)

**Comment by adamdruppe** [comment]
You want that "updated rows" thing? The C function is PQcmdTuples but indeed I never added that to my interface.

Just pushed something to master. It is a bit awkward to use but it is like:

```
       auto pg = new PostgreSql("dbname=test");
        auto res = cast(PostgresResult) pg.query("UPDATE employees set name = 'aas' where id = 1749411001000");
        assert(res !is null);
        import std.stdio;
        writeln(res.affectedRows);
```

Notice the cast so you can get to the new member.

I'll see about cleaning up the interface later.

**Comment by AlexanderZhirov** [comment]
> You want that "updated rows" thing?

Yes, I would like to get the result of the number of affected rows after the update.

**Comment by adamdruppe** [comment]
that function i don't think tells you which ones actually changed but rather which ones matched the where clause

but yeah it seems to work so if that master thing works let me know
